CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## [Unreleased]
 
 ### BREAKING CHANGES
+- Instances of `@cumulus/ingest/SftpProviderClient` no longer implicitly connect
+  when `download`, `list`, or `sync` are called. You must call `connect` on the
+  provider client before issuing one of those calls. Failure to do so will
+  result in a "Client not connected" exception being thrown.
+- Instances of `@cumulus/ingest/SftpProviderClient` no longer implicitly
+  disconnect from the SFTP server when `list` is called.
+- Instances of `@cumulus/sftp-client/SftpClient` must now be expclicitly closed
+  by calling `.end()`
+- Instances of `@cumulus/sftp-client/SftpClient` no longer implicitly connect to
+  the server when `download`, `unlink`, `syncToS3`, `syncFromS3`, and `list` are
+  called. You must explicitly call `connect` before calling one of those
+  methods.
+- Instances of `@cumulus/sftp-client/SftpClient` must now be explicitly closed
```

Comment:
I think this note is in here twice

Reply:
c2f282ae60832876d4a69ed689d976b9c4926da3